src/lib.rs

```diff
@@ -699,14 +699,10 @@ pub unsafe extern "C" fn wgpuAdapterGetInfo(
         Err(err) => handle_error_fatal(err, "wgpuAdapterGetInfo"),
     };
 
-    info.vendor = CString::new(format!("{:#x}", result.vendor))
-        .unwrap()
-        .into_raw();
+    info.vendor = CString::new(result.driver).unwrap().into_raw();
     info.architecture = CString::default().into_raw();
```

Comment:
```suggestion
    info.architecture = CString::default().into_raw(); // TODO(webgpu.h)
```
Let also ressurect the `TODO` comment for the `architecture`, from https://github.com/gfx-rs/wgpu-native/blob/a1ede115d0d669b88b34447e58752335cb882e61/src/lib.rs#L715